lib/View.js

```diff
@@ -31,10 +42,8 @@ class View extends Node {
 
   dimension ({ cubeDimension }) {
     if (cubeDimension) {
-      const term = toTerm(cubeDimension)
```

Comment:
fix to bug reported by martin (in cube-viewer code): Must compare both by their path.